src/GRA.Controllers/ViewModel/Profile/BadgeListViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
```

Comment:
Clear unnecessary usings.

Reply:
This one is used for line 9: `public ICollection<UserLog> UserLogs { get; set; }`